### DISCUSSION TITLE
Apple raccourci WebSSH command launch

### DISCUSSION THREAD

**Discussion by Fan59** [isontheline/pro.webssh.net#770] [Q&A]
Hello,
First really thank’s for this great app, i have a question,
Is it possible. To launch with apple raccourci application a ssh connexion with command the the destination server (like snippet script) but with one click button 

Best regards
François

**Comment by isontheline** [comment]
Bonjour @Fan59 François 👋

Thank you a lot for your support about WebSSH 🙏

Unfortunately, these task aren't - yet - implemented inside WebSSH :
* https://github.com/isontheline/pro.webssh.net/issues/354
* https://github.com/isontheline/pro.webssh.net/issues/650

Belle journée à vous
